cmake/OpenCVDetectPython.cmake

```diff
@@ -88,6 +88,9 @@ function(find_python preferred_version min_version library_env include_dir_env
         find_host_package(PythonLibs "${_version_major_minor}")
       else()
         find_host_package(PythonLibs "${_version_major_minor}.${_version_patch}" EXACT)
+        if(NOT PYTHONLIBS_FOUND)
```

Comment:
the proposed fix does not help :( and how come my solution break python2 if it calls find_host_package another time only if pythonlibs are not found?